firestore.rules

```diff
@@ -1,13 +1,53 @@
 rules_version = '2';
 service cloud.firestore {
   match /databases/{database}/documents {
+    function validAmount(docData){
+      return docData.amount is map 
+      && docData.amount.type in ['UNKNOWN', 'FIXED','VARIES','FULL_TUITION']
+      && docData.amount.min is int
+      && docData.amount.max is int
+    }
+
+    function validReqs(docData){
+      return !('requirements' in docData) 
```

Comment:
Might be nice to ensure this is always set even if empty (though not sure of current db state).

Reply:
I check the db and the one that were added via google sheets don't have a`requirements` field

Reply:
Ah right, I did it that way so that when we actually edit them we remember to set the requirements. Alright in a few weeks then.